superheroes.yaml

```diff
@@ -150,6 +153,9 @@ scripts:
       - regex: (?<HOST.HEROES_REST_POD_ID>[a-f0-9]{64}$)
         else:
         - abort: failed to capture pod id for Heroes rest service
+      - sh: ${{CONTAINER_RUNTIME}} inspect --format='{{.State.Pid}}' ${{HOST.HEROES_REST_POD_ID}}
+        then:
+        - set-state: HOST.HEROES_REST_PID
```

Comment:
If the container has a wrapper script, does this get the PID of the java process, or the wrapper script?
The semeru builds use a wrapper script to start the java process inside the container

Reply:
Good question, I don't know.. I will check it

Reply:
Ok, I just tried a simple example and `inspect --format='{{.State.Pid}}'` will get the main process PID, i.e., the wrapper in the case of semeru builds 🫤

Reply:
With a dirty trick https://github.com/RedHatPerf/superheroes-automation/pull/9/commits/51c64463891a0bc9a4ad7237e201770c015f6837 I was able to retrieve the correct pid when using semeru builds - looks working but it is assuming the main command is a `java` command which should be always true for semeru builds